indico/web/client/js/legacy/libs/timetable/Draw.js

```diff
@@ -273,7 +253,7 @@ type(
 
       this.leftPos = leftPos;
       this.width = width;
-      this.height = this.blockData.end - this.blockData.start - 3;
+      this.height = this.blockData.end - this.blockData.start - 2;
```

Comment:
what is this mysterious `2`? :D

Reply:
You're right, it's not self-explanatory, had the same question before. This `2` is an extra padding to prevent block entries from being too close together.

I'm adding a comment referring such.

![image](https://user-images.githubusercontent.com/12183954/77157581-91fc6800-6a99-11ea-877b-2861f47f8fb8.png)
![image](https://user-images.githubusercontent.com/12183954/77158348-0b488a80-6a9b-11ea-8cbc-852218f18732.png)

Reply:
Ok, thanks!